docs/metrics.md

```diff
@@ -101,6 +104,111 @@ EOE
+        unit: ms  # unit of metric
+        dimensions: [ModelName, host]  # dimensions of metric (referenced from the above dimensions dict)
+  gauge:
+    - NameOfGaugeMetric:
+        unit: ms
+        dimensions: [ModelName, host]
+  histogram:
+    - NameOfHistogramMetric:
+        unit: ms
+        dimensions: [ModelName, host]
+
+model_metrics:  # backend metrics
+  counter:  # metric type
+    - InferenceTimeInMS:  # name of metric
+        unit: ms  # unit of metric
+        dimensions: [ModelName, Level]  # dimensions of metric (referenced from the above dimensions dict)
+    - NumberOfMetrics:
+        unit: count
+        dimensions: [model_name]
+  gauge:
+    - GaugeModelMetricNameExample:
+        unit: ms
+        dimensions: [ModelName, Level]
+  histogram:
+    - HistogramModelMetricNameExample:
+        unit: ms
+        dimensions: [ModelName, Level]
+```
+
+
+These are the default metrics within the yaml file, but the user can either delete them to their liking / ignore them altogether, because these metrics will not be emitted
+unless they are edited.
+
+
+### How it works
+
+Whenever torchserve starts, the [model_loader](https://github.com/pytorch/serve/blob/master/ts/model_loader.py) initializes
+`service.context.metrics` with the [MetricsCache](https://github.com/pytorch/serve/blob/master/ts/metrics/metric_cache_yaml.py) object.
+The `model_metrics` (backend metrics) section within the specified yaml file will be parsed,
+and Metric objects will be created based on the parsed section and added  that are added to the cache.
+
+This is all done internally, so the user does not have to do anything other than specifying the desired yaml file.
+
+*Users have the ability to parse other sections of the yaml file manually, but the primary purpose of this functionality is to
+parse the backend metrics from the yaml file.
+
+### User Manual - starting TorchServe with a yaml file specified
+
+1. Create a `metrics.yaml` file to parse metrics from OR utilize default [metrics_default.yaml](https://github.com/pytorch/serve/blob/master/frontend/server/src/test/resources/metrics_default.yaml)
+
+
+2. Set `metrics_config` argument equal to the yaml file path in the `config.properties` being used:
+    ```properties
+    ...
+    ...
+    # enable_metrics_api=false
+    workflow_store=../archive/src/test/resources/workflows
+    metrics_config=../../../ts/tests/unit_tests/metrics_yaml_testing/metrics.yaml
+    ...
+    ...
+    ```
+
+   If a `metrics_config` argument is not specified, the default yaml file will be used.
+
+
+3. Run torchserve and specify the path of the `config.properties` after the `ts-config` flag:
+   (example using [Huggingface_Transformers](https://github.com/pytorch/serve/tree/master/examples/Huggingface_Transformers))
+
+   ```torchserve --start --model-store model_store --models my_tc=BERTSeqClassification.mar --ncs --ts-config ../../frontend/server/src/test/resources/config.properties```
+
```

Comment:
should mention that logs are visible in `logs/model_metrics.log` although when I just downloaded the densenet.mar file we have in our getting started guide I didn't see anything populated there - was hoping to see some default metrics